src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -961,6 +961,15 @@ theorem not_not {b : BitVec w} : ~~~(~~~b) = b := by
   ext i
   simp
 
+theorem not_eq_iff_eq_not {x y : BitVec w} : ~~~ x = y ↔ x = ~~~ y := by
```

Comment:
```suggestion
theorem not_eq_comm {x y : BitVec w} : ~~~ x = y ↔ x = ~~~ y := by
```

This seems to be the spelling that is used for `Int`s: `Int.neg_eq_comm {a b : Int} : -a = b ↔ -b = a`